SU2_CFD/src/numerics/NEMO/convection/ausm_slau.cpp

```diff
@@ -482,6 +482,88 @@ void CUpwAUSM_NEMO::ComputeInterfaceQuantities(const CConfig* config, su2double*
   for (auto iDim = 0ul; iDim < nDim; iDim++) pressure[iDim] = (P_LP + P_RM) * UnitNormal[iDim];
 }
 
+CUpwAUSMPLUSUP_NEMO::CUpwAUSMPLUSUP_NEMO(unsigned short val_nDim, unsigned short val_nVar,
+                                           unsigned short val_nPrimVar, unsigned short val_nPrimVarGrad,
+                                           const CConfig* config)
+    : CUpwAUSM_SLAU_Base_NEMO(val_nDim, val_nVar, val_nPrimVar, val_nPrimVarGrad, config) {
+
+  Minf = config->GetMach();
+  Kp = 0.25;
+  Ku = 0.75;
+  sigma = 1.0;
+
+  if (Minf < EPS)
+    SU2_MPI::Error("AUSM+Up requires a reference Mach number (\"MACH_NUMBER\") greater than 0.", CURRENT_FUNCTION);
+}
+
+void CUpwAUSMPLUSUP_NEMO::ComputeInterfaceQuantities(const CConfig* config, su2double* pressure,
+                                                    su2double& interface_mach, su2double* interface_soundspeed) {
+
+  /*--- Compute interface speed of sound (aF) ---*/
+
+  const su2double astarL = sqrt(2.0*(Gamma-1.0)/(Gamma+1.0)*Enthalpy_i);
+  const su2double astarR = sqrt(2.0*(Gamma-1.0)/(Gamma+1.0)*Enthalpy_j);
```

Comment:
Just out of curiosity is this still accurate for NEMO?

Reply:
Probably not exactly, though we I need to make some changes so we include local gamma values (Gamma_i and j).